ansible/roles/set-repositories/tasks/rhn-repos.yml

```diff
@@ -18,7 +18,11 @@
     pool_ids: "{{ rhsm_pool_ids }}"
     auto_attach: false
 
-- name: "Run 'subscription-manager to disable/enable repos"
+- name: "Run subscription-manager to disable all repos"
```

Comment:
The changes in the Jim/agnosticd branch are from an older version of upstream. However, do we want to include this disable task? It appears to have been dropped.

There may be differences with the upstream usage of satellite as well.